Automattic/wp-calypso
# [ETK][error-reporting] Update Sentry packages

#### Changes proposed in this Pull Request

* Update Sentry npm packages to their latest versions.

#### Testing instructions

Build and sync this version of ETK to your sandbox and trigger an error manually. Just fire-up the console and run this JS snippet:

```
var foo = document.createElement('script');
foo.innerHTML = "throw new Error('hi there!');"
document.body.appendChild(foo);
```

You should see the error in Sentry.
